src/app/components/list/list.js

```diff
@@ -122,8 +121,6 @@ listControllers.controller('ListCtrl', ['$scope', '$routeParams', '$location', '
       });
     };
     $scope.list.$promise.then(listCallback);
-    $scope.list.$httpPromise.then(listCallback);
```

Comment:
Ok I reread the documentation of angular-cached-resource and you're right, this can be removed, because $promise.then will be called no matter what (even if the list is not cached)